paper.md

```diff
@@ -328,11 +328,56 @@ From Table I & Figure III, it can be seen how the portability of Popper workflow
+For an user whose primary goal is to automate running a set of containerized scripts in sequence, learning these new workflow languages and syntaxes might add to the overall complexity.
+One of Popper's primary goals is to minimize the workflow language overhead as much as possible by allowing users to specify workflows using vanilla YAML syntax, thus keeping the learning curve flat and preventing sources of confusion.
+
+## Workflow execution engines
+
+Workflow execution engines can be categorized in several different categories.
+In this section, we have discussed three frequently used catergories of workflow execution engines namely Generic, Cloud Native and Container Native and compared their pros and cons with Popper.
+
+### Generic workflow execution engines
+
+Let us take a relatively simple scenario where users have a list of single purpose tasks as scripts and they want to automate running them in containers in some sequence.
+In order to accomplish this goal of running a list of containerized tasks using existing workflow engines, users need to learn a specific workflow language, deploy a workflow engine service and learn executing workflows on this service.
+These tasks may not be always trivial to accomplish, if we assume the only thing users should care about is writing experimentation scripts and running them inside containers.
+Few examples of this category are stable and mature scientific workflow engines like Nextflow, Pegasus and Taverna which have recently introduced native support running steps in software containers;
+Industry standard workflow engines like Apache Airflow and Luigi [@_luigi_] which require specifying workflows using programming languages and also provide pluggable interfaces that require the installation of separate plugins;
+For example, Airflow and Luigi uses Python, Copper use Java, Dagr use Scala and SciPipe use Go as their workflow definition language.
+The goal with Popper is to minimize overhead both in terms of workflow language syntax and infrastructural requirements for running workflows and hence allow users to focus solely on writing the workflows.
+The first issue is already addressed in the previous subsection, but it's also relevant here because not all engines support standard workflow languages such as CWL and also learning specific programming languages for the purpose of workflow execution seems like an exaggeration.
+Most of these standard workflow engines like Pegasus, Airflow and Luigi also require a standalone service that users need to learn how to deploy and interact with before executing workflows, thus adding to the complexity.
+Popper also mitigates this issue as it can be downloaded and run as a standalone executable and does not assume any service deployment or infrastructural management prior to running workflows.
+
+### Cloud native workflow execution engines
```

Comment:
since "cloud native" is "container native" as well, the order of these two subsections could be swapped. And this distinction can also be made at the beginning of the "cloud native" section in order to minimize confusion